technical_architecture/best_practices/reusable_bundle.rst

```diff
@@ -56,8 +56,8 @@ You can find some configuration examples on our `Akeneo-Labs`_ projects (`scruti
 .. _Travis: https://travis-ci.org/
 .. _Scrutinizer: https://scrutinizer-ci.com/
 .. _Akeneo-Labs: https://github.com/akeneo-labs
-.. _scrutinizer.yml: https://github.com/akeneo-labs/CustomEntityBundle/blob/master/.scrutinizer.yml
-.. _travis.yml: https://github.com/akeneo-labs/CustomEntityBundle/blob/master/.travis.yml
+.. _scrutinizer.yml: https://github.com/akeneo-labs/CustomEntityBundle/blob/3.0/.scrutinizer.yml
```

Comment:
not sure they follow the same version
check with JM

Reply:
Seen with him already ;)